t/node/chash-hashon.t

```diff
@@ -625,3 +625,90 @@ chash_key: "chash_val_3"
 chash_key: "chash_val_4"
 chash_key: "chash_val_5"
 chash_key: "chash_val_6"
+
+=== TEST 13: set route(two upstream node, type chash), hash_on vars_combinations
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "uri": "/server_port",
+                    "upstream": {
+                        "key": "$http_custom_header-$http_custom_header_second",
+                        "type": "chash",
+                        "hash_on": "vars_combinations",
+                        "nodes": {
+                            "127.0.0.1:1980": 1,
+                            "127.0.0.1:1981": 1
+                        }
+                    }
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- no_error_log
+[error]
+
```

Comment:
Style: need three blank lines.

Reply:
@Firstsawyou here is the correct way: https://github.com/apache/apisix/pull/3278#discussion_r557028364